test-suite/src/test/scala/org/scalajs/testsuite/javalib/ConcurrentLinkedQueueTest.scala

```diff
@@ -0,0 +1,183 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2015, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package  org.scalajs.testsuite.javalib
+
+import scala.language.implicitConversions
+
+import scala.collection.JavaConversions._
+
+import org.scalajs.jasminetest.JasmineTest
+
+import scala.scalajs.runtime.UndefinedBehaviorError
+
+import java.util.concurrent.ConcurrentLinkedQueue
+
+object ConcurrentLinkedQueueTest extends ConcurrentLinkedQueueTest(new ConcurrentLinkedQueueFactory)
+
+abstract class ConcurrentLinkedQueueTest[F <: ConcurrentLinkedQueueFactory](listFactory: F) extends AbstractCollectionTest(listFactory) {
```

Comment:
This line is too long. It won't pass scalastyle. Break the line before `listFactory`, with 4 spaces.